contracts/root/withdrawManager/WithdrawManagerStorage.sol

```diff
@@ -65,7 +66,7 @@ contract WithdrawManagerStorage is ProxyStorage, WithdrawManagerHeader {
     // uint32 constant internal ITERATION_GAS = 52000;
 
     // So putting an upper limit of 155000 + 52000 + leeway
-    uint32 internal constant ON_FINALIZE_GAS_LIMIT = 210000;
+    uint32 internal constant ON_FINALIZE_GAS_LIMIT = 300000;
```

Comment:
We should keep this var updatable so that we can handle Istanbul like incidents.

Reply:
or at least please don't make it `constant` so that we can upgrade proxy and add a function to update this.